dev-python/confuse/confuse-1.0.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+inherit distutils-r1
+
+DESCRIPTION="Confuse is a configuration library for Python that uses YAML"
+HOMEPAGE="https://github.com/beetbox/confuse"
+SRC_URI="https://github.com/beetbox/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+BDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? ( dev-python/nose[${PYTHON_USEDEP}] )
+"
+DEPEND="
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+"
+RDEPEND="${DEPEND}"
+
+python_compile_all() {
+	use doc && emake -j1 -C docs html
```

Comment:
I took the `-j1` limitation out and it worked fine. `-j24` FYI.

Reply:
Removed :)